roles/pulp_common/templates/settings.py.j2

```diff
@@ -1,3 +1,3 @@
 {% for setting, value in __pulp_common_merged_pulp_settings.items() %}
-{{ setting | upper }} = {{ value | string | capitalize if value | string in ["True", "False"] else value | to_json }}
```

Comment:
Remind me again, but shouldn't the repr filter be an entirely different change from the rest of the PR that focuses on testing pulp installed as a collection?

Reply:
Actually this is the meat of the PR. The rest is a way to make the tests pass.

Reply:
About to split the PR, as the "make the ci pass again" got way out of scope.